tensorboard/uploader/uploader_subcommand.py

```diff
@@ -462,12 +465,23 @@ def execute(self, server_info, channel):
         except KeyboardInterrupt:
             interrupted = True
         finally:
+            if self.one_shot and not uploader.tracker.has_data():
+                print(
+                    "Tensorboard was run in `one_shot` mode, but did not find "
+                    "any uploadable data in the specified logdir: %s\n"
+                    "An empty experiment was created. "
+                    "To delete the empty experiment execute the following\n\n"
```

Comment:
I'd suggest "To delete the empty experiment you can execute the following". It is a bit more friendly IMO.